docs/pages/push-notifications/push-notifications-setup.mdx

```diff
@@ -1,28 +1,79 @@
 ---
-title: Push Notifications Setup
+title: Push notifications setup
 sidebar_title: Setup
 ---
 
 import { Tab, Tabs } from '~/ui/components/Tabs';
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { BoxLink } from '~/ui/components/BoxLink';
 
-To get the client-side ready for push notifications, the 2 main things we need are:
+To use Expo's push notification service, you need to configure your app to install a set of libraries, add functions to handle notifications and configure credentials for Android and iOS. After going through these steps, you'll be able to test notification sending and receiving notifications on a device.
+
+To get the client-side ready for push notifications, the following is required:
 
 - The user's permission to send them push notifications
-- The user's ExpoPushToken- if push notifications are mail, then the ExpoPushToken is the user's address.
+- The user's [`ExpoPushToken`](/versions/latest/sdk/notifications/#expopushtoken) &mdash; if push notifications are mail, then the `ExpoPushToken` is the user's address.
```

Comment:
```suggestion
- The user's [`ExpoPushToken`](/versions/latest/sdk/notifications/#expopushtoken).
```
We can keep this extra copy if you'd like, but I don't think it's totally needed.